metadata-models/src/main/pegasus/com/linkedin/ml/metadata/MLFeatureTableProperties.pdl

```diff
@@ -1,14 +1,15 @@
 namespace com.linkedin.ml.metadata
 
 import com.linkedin.common.Urn
+import com.linkedin.common.CustomProperties
 
 /**
  * Properties associated with a MLFeatureTable
  */
 @Aspect = {
   "name": "mlFeatureTableProperties"
 }
-record MLFeatureTableProperties {
+record MLFeatureTableProperties includes CustomProperties {
```

Comment:
You should add CustomProperties to the aspect union

Reply:
Punting this for now until we fix CustomProperties in general